content/webapp/pages/events/[eventId]/index.tsx

```diff
@@ -415,14 +416,15 @@ const EventPage: NextPage<EventProps> = ({
               .concat(event.policies)
               .concat(
                 event.interpretations.map(interpretation => {
-                  const iconName = camelize(
-                    interpretation.interpretationType.title
-                  );
-
                   const description = getDescription(interpretation);
-
+                  const matchingInterpretation = interpretationTypeIconMap.find(
+                    item =>
+                      item.prismicId === interpretation.interpretationType.id
+                  );
+                  const iconName =
+                    matchingInterpretation?.iconName || undefined;
```

Comment:
np `matchingInterpretation?.iconName` should return `undefined` anyway so could possibly just use that?